science/hacktv/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+PortGroup           github 1.0
+
+github.setup        kesterlester hacktv 0.0.4 v
+github.tarball_from releases
+
+categories          science
+platforms           darwin
+license             GPL-3+
+maintainers         kesterlester
```

Comment:
Can you follow the maintainers keyword format?

https://guide.macports.org/#reference.keywords.maintainers

Reply:
Fixed in 068e26a

Reply:
not completely, it should include both an e-mail address and a GitHub handle; additionally, we strongly prefer to also add `openmaintainer` such that minor changes to the Portfile can be made without your explicit approval

Reply:
Hopefully ce99322956b9464d5eb383b6cb71b6cc58ee35a5 now solves this part. Still working on other parts.